docs/Users_Guide/scatter.rst

```diff
@@ -0,0 +1,227 @@
+************
+Scatter Plot
+************
+
+Description
+===========
+The scatter plot is useful for illustrating relationships between pairs of continuous variables.
+This plot was developed to support plotting MPR (matched pair) data from the MET point-stat tool.
+**NOTE:** This MET output data must first be reformatted into a format that can be read in by the scatter plot code.
+This reformatting was accomplished through the METdataio METreformat module. The reformatted data
+consists solely of MPR linetype data and all the column headers are labelled according to the
+MPR linetype column names specified in the MET User's Guide, Table 11.20 (Point-stat tool).
```

Comment:
```suggestion
MPR linetype column names specified in the Point-Stat section of the MET User's Guide in `Table 11.20 <https://metplus.readthedocs.io/projects/met/en/latest/Users_Guide/point-stat.html#table-ps-format-info-seeps>`_ "Format information for SEEPS (Stable Equitable Error in Probability Space) output line type".
```
Suggest adding a link and more information, like the table name, in case the link should break, which hopefully it wouldn't, but just in case.